### ISSUE TITLE
Screen reading is not reading the columns headers name when focus is on the table cells even though scrollable and navigatable property to true

### ISSUE THREAD

**Issue by jykukrej** [telerik/kendo-ui-core#5576]
**Bug report**

Screen reading is not reading the columns headers name when focus is on the table cells even though scrollable and navigatable property set to true.

**Reproduction of the problem**

https://demos.telerik.com/kendo-ui/grid/index
Screen reader is not able to read the header when the focus is on table content.
We tried by removing the scrollable ( or set to false) property and then observed that screen reader is able to read the corresponding column header.

Based on the feedback given in this issue :[](https://github.com/telerik/kendo-ui-core/issues/5518) scrollable should always be set to true but we have seen that in html code "aria-describedby" is not added.

**Expected/desired behavior**

Screen reader should the column headers title when focus is on the table content.

**Environment**
Kendo UI version:Kendo U1R1 2020
Browser: [all]

**Comment by dmanova** [comment]
The attribute `aria-describedby` is added to the table cell tag `< td >` in case the Kendo UI Grid has the configuration `navigatable: true`. Then when the screen reader, reads the information about the current cell it uses `aria-describedby` value to link with the corresponding column header and reads it. 

Check this Dojo sample that illustrates the case: https://dojo.telerik.com/AHopUduN

**Comment by dmanova** [comment]
@jykukrej  have you check the suggested configuration in the previous response? Do the screen readers read the columns' headers now?

**Comment by ylux587** [comment]
Hi, We tried on the solution suggested but still it is not working. The column header does not get read by Narrator. See attached zip file
[demo_.zip](https://github.com/telerik/kendo-ui-core/files/5220106/demo_.zip)

**Comment by dmanova** [comment]
Hi, @ylux587 and @jykukrej, I find a configuration at Narrator that could be the reason for this behavior. What makes the difference in reading the grid's columns headers is the checkbox with text "Hear advanced detail, like help text on buttons and other controls". 

Please confirm if you hear the column title with that option selected.

Here is a screenshot of Narrator's settings:
![image](https://user-images.githubusercontent.com/52446546/93811534-f242b100-fc58-11ea-8609-82404d9b7f33.png)

**Comment by dmanova** [comment]
The issue is related to the screen reader configuration. Check the previous comments.

**Comment by booninja** [comment]
This is still an issue for NVDA screen readers. Both the sample from telerik and our implementation.

Here is a snippit from the NVDA speech viewer when scanning through the telerik sample:

Category: Seafood, Total Sales: $21,100.00
row 2 column 1
column 2 check box not checked Select row
column 3 Ikura
column 4 $31.00
column 5 not available
column 6 Seafood
column 7 slider 2
column 8 graphic Great Britain

As can be seen, the screen reader is ignoring the column headers.

**Comment by booninja** [comment]
After some looking around, it seems to be that NVDA is programmed to not read any aria attributes on tables, as it is specifically programmed to read tables in certain way. (found here: https://github.com/nvaccess/nvda/issues/5567)
Because the kendo header columns are in a seperate table, NVDA cannot connect the cells to their columns. 

Seeing how the column header is important to give the user context, I have found this solution: 
```
<th id="subject_label"> Subject </th>
```
```
<td class="" role="gridcell" id="paginggridview-case_active_cell">
        <span aria-labelledby="subject_label"></span> //The span will not be ignored by NVDA 
         11/11/2021
</td>
```
And for cells with links: 
```
<td role="gridcell">
    //Links are also not ignored by NVDA 
     <a href=""  id="subject_value" aria-labelledby="subject_label subject_value">Subject cell</a> 
</td>
```

There is probably a better way to go about this, but it works. I would be happy to see any better solutions. :)

**Comment by GrooyaO** [comment]
> After some looking around, it seems to be that NVDA is programmed to not read any aria attributes on tables, as it is specifically programmed to read tables in certain way. (found here: [nvaccess/nvda#5567](https://github.com/nvaccess/nvda/issues/5567)) Because the kendo header columns are in a seperate table, NVDA cannot connect the cells to their columns.
> 
> Seeing how the column header is important to give the user context, I have found this solution:
> 
> ```
> <th id="subject_label"> Subject </th>
> ```
> 
> ```
> <td class="" role="gridcell" id="paginggridview-case_active_cell">
>         <span aria-labelledby="subject_label"></span> //The span will not be ignored by NVDA 
>          11/11/2021
> </td>
> ```
> 
> And for cells with links:
> 
> ```
> <td role="gridcell">
>     //Links are also not ignored by NVDA 
>      <a href=""  id="subject_value" aria-labelledby="subject_label subject_value">Subject cell</a> 
> </td>
> ```
> 
> There is probably a better way to go about this, but it works. I would be happy to see any better solutions. :)

Do you by any chance have a code snippet for this?

**Comment by MilenaCh** [comment]
The relation between Grid's header and table cells has been improved in our latest releases. I've tested [Grid Overview demo ](https://demos.telerik.com/kendo-ui/grid/index) with NVDA and the column name is announced properly when focus a table cell. Can you check the behavior on your end with the latest version?

**Comment by GrooyaO** [comment]
Currently having issues with deep-equal library when updating to new v. I saw some webpack fixes but we only have gulp.
